.github/workflows/integration-workflow.yml

```diff
@@ -2,6 +2,7 @@ on:
   push:
     branches:
     - master
+    - mael/linker-refactoring
```

Comment:
Should it be there?

Reply:
Nope, will remove it, I just worked on a separate branch where I needed tests to run.